crates/nu-command/src/filters/par_each.rs

```diff
@@ -49,8 +47,30 @@ impl Command for ParEach {
                 result: None,
             },
             Example {
-                example: r#"[1 2 3] | par-each -n { |it| if $it.item == 2 { $"found 2 at ($it.index)!"} }"#,
-                description: "Iterate over each element, print the matching value and its index",
+                example: r#"[foo bar baz] | par-each {|e| $e + '!' } | sort"#,
+                description: "Output can still be sorted afterward",
+                result: Some(Value::List {
+                    vals: vec![
+                        Value::test_string("bar!"),
+                        Value::test_string("baz!"),
+                        Value::test_string("foo!"),
+                    ],
+                    span: Span::test_data(),
+                }),
+            },
+            // I'm not sure why this doesn't work.
+            // Example {
```

Comment:
It would be nice to have this test working but I couldn't figure it out. I tried several different iterations without success.

Reply:
A shame, as I feel like that one offers a good strategy for using par-each in order-sensitive situations.

Reply:
Let me play some more. At the end of the day, the code snippet works in nushell. It's just that the test doesn't quite work right. So, we could just have a `result: None` with the code snippet in there.

This is what it returns.
```
---- filters::par_each::test::test_examples stdout ----
thread 'filters::par_each::test::test_examples' panicked at 'assertion failed: `(left == right)`
  left: `Error { error: CantFindColumn("item", Span { start: 128, end: 132 }, Span { start: 124, end: 127 }) }`,
 right: `List { vals: [Int { val: 2, span: Span { start: 0, end: 0 } }, Int { val: 4, span: Span { start: 0, end: 0 } }, Int { val: 6, span: Span { start: 0, end: 0 } }], span: Span { start: 0, end: 0 } }`: The example result differs from the expected value', crates\nu-command\src\example_test.rs:231:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    filters::par_each::test::test_examples

test result: FAILED. 398 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 1.27s

error: test failed, to rerun pass `-p nu-command --lib`
```

Reply:
I think I figured it out. Some commands weren't in `example_test.rs`.